doc/modules/qualitymetrics/isi_violations.rst

```diff
@@ -10,40 +10,37 @@ Calculation
 Neurons have a refractory period after a spiking event during which they cannot fire again.
 Inter-spike-interval (ISI) violations refers to the rate of refractory period violations (as described by [Hill]_).
 
+We aim to calculate the contamination rate :math:`C`, measuring the ratio of isi violations in the spike-train of a unit.
 The calculation works under the assumption that the contaminant events happen randomly or come from another neuron that is not correlated with our unit.
 A correlation will lead to an overestimation of the contamination, whereas an anti-correlation will lead to an underestimation.
 
-Different formulas have been developed over the years.
+Different formulas have been developed, but all require:
 
-Calculation from the [Hill]_ paper
-----------------------------------
+- :math:`T` the duration of the recording.
+- :math:`N` the number of spikes in the unit's spike train.
+- :math:`t_r` the duration of the unit's refractory period.
+- :math:`n_v` the number of violations of the refractory period.
 
-The following quantities are required:
+Calculation from the [UMS]_ package
```

Comment:
Is it worth somewhere directly linking to the relevant [line on their github](https://github.com/danamics/UMS2K/blob/07e99b11e12a874d044e1367e548baeeb63e507b/quality_measures/rpv_contamination.m#L75)?

Reply:
I think a link to the matlab file is good, since the file only contains this calculation (and explains the calculation!)

Reply:
Oh I didn't see that, yes sounds good!